app/controllers/application_controller.rb

```diff
@@ -25,10 +25,6 @@ def livestream_editor?
     current_user.has_permission? "Livestream editor"
   end
 
-  def unreleased_feature_user?
-    current_user.has_permission? "Unreleased feature"
```

Comment:
I was wondering about either leaving this in, or coming up with some agreed thing to use across apps so that we don't proliferate different versions of this?

Reply:
Ah so leave it in the controller for future prototypes - yeah good idea.